mindsdb/api/executor/datahub/datanodes/mindsdb_tables.py

```diff
@@ -0,0 +1,418 @@
+                    val.get("type"),
+                    val.get("title"),
+                    val.get("description"),
+                    val.get("version"),
+                    connection_args,
+                    import_success,
+                    import_error,
+                ]
+            )
+
+        df = pd.DataFrame(data, columns=self.columns)
+        return df
+
+
+class JobsTable(MdbTable):
+    name = "JOBS"
+    columns = [
+        "NAME",
+        "PROJECT",
+        "START_AT",
+        "END_AT",
+        "NEXT_RUN_AT",
+        "SCHEDULE_STR",
+        "QUERY",
+        "IF_QUERY",
+        "VARIABLES",
+    ]
+
+    def get_data(self, query: ASTNode = None, **kwargs):
+        jobs_controller = JobsController()
+
+        project_name = None
+        if (
+            isinstance(query, Select)
+            and type(query.where) is BinaryOperation
+            and query.where.op == "="
+            and query.where.args[0].parts == ["project"]
+            and isinstance(query.where.args[1], Constant)
+        ):
+            project_name = query.where.args[1].value
+
+        data = jobs_controller.get_list(project_name)
+
+        columns = self.columns
+        columns_lower = [col.lower() for col in columns]
+
+        # to list of lists
+        data = [[row[k] for k in columns_lower] for row in data]
+
+        return pd.DataFrame(data, columns=columns)
+
+
+class TriggersTable(MdbTable):
+    name = "TRIGGERS"
+    columns = [
+        "TRIGGER_CATALOG",
+        "TRIGGER_SCHEMA",
+        "TRIGGER_NAME",
+        "EVENT_MANIPULATION",
+        "EVENT_OBJECT_CATALOG",
+        "EVENT_OBJECT_SCHEMA",
+        "EVENT_OBJECT_TABLE",
+        "ACTION_ORDER",
+        "ACTION_CONDITION",
+        "ACTION_STATEMENT",
+        "ACTION_ORIENTATION",
+        "ACTION_TIMING",
+        "ACTION_REFERENCE_OLD_TABLE",
+        "ACTION_REFERENCE_NEW_TABLE",
+        "ACTION_REFERENCE_OLD_ROW",
+        "ACTION_REFERENCE_NEW_ROW",
+        "CREATED",
+        "SQL_MODE",
+        "DEFINER",
+        "CHARACTER_SET_CLIENT",
+        "COLLATION_CONNECTION",
+        "DATABASE_COLLATION",
+    ]
+
+    mindsdb_columns = ["NAME", "PROJECT", "DATABASE", "TABLE", "QUERY", "LAST_ERROR"]
+
+    def get_data(self, query: ASTNode = None, inf_schema=None, **kwargs):
+        from mindsdb.interfaces.triggers.triggers_controller import TriggersController
+
+        triggers_controller = TriggersController()
+
+        project_name = None
+        if (
+            isinstance(query, Select)
+            and type(query.where) is BinaryOperation
+            and query.where.op == "="
+            and query.where.args[0].parts == ["project"]
+            and isinstance(query.where.args[1], Constant)
+        ):
+            project_name = query.where.args[1].value
+
+        data = triggers_controller.get_list(project_name)
+
+        columns = self.mindsdb_columns
+        if inf_schema.session.api_type == 'sql':
```

Comment:
Do we need `api_type` arg only for this specific case?
 Is it a good idea to return a different set of columns for each API?

Reply:
yes, only for this case. 
we use the whole set columns for db managers (like dbeaver) and limited set for web GUI, to make able user see only essential information

Also division between system_tables and mindsdb_tables - it is only for SQL server, to make sql clients working. In GUI we show only mindsb tables (though allow to select from all of them)